plugins/module_utils/vmware.py

```diff
@@ -855,14 +855,16 @@ def is_truthy(value):
 
 # options is the dict as defined in the module parameters, current_options is
 # the list of the currently set options as returned by the vSphere API.
-def option_diff(options, current_options):
+# When truthy_strings_as_bool is True, strings like 'true', 'off' or 'yes'
+# are converted to booleans.
+def option_diff(options, current_options, truthy_strings_as_bool=True):
```

Comment:
`vmware_cluster_drs` also uses `option_diff`. Should we turn `truthy_strings_as_bool` false there too?

Reply:
@goneri I'm not sure... you see, we don't have any known issues for `vmware_cluster_drs` that are related. Changing the current behavior might break something. In case it turns out that `vmware_cluster_drs` has the same problem, the fix would be quite easy. But I didn't want to "fix" a problem in advance that maybe doesn't exist and possibly break things.

Reply:
Thank you @mariolenz for the answer.